js/controller/BoardController.js

```diff
@@ -361,4 +384,6 @@ app.controller('BoardController', function ($rootScope, $scope, $stateParams, St
 		return card.attachmentCount;
 	};
 
+	$scope.loadDeletedEntity(CardService, 'deletedCards');
```

Comment:
Do you load deleted entries in general? I think we should only load those once the user opens the sidebar.